integration_tests/__tests__/failures-test.js

```diff
@@ -11,8 +11,33 @@
 const path = require('path');
 const runJest = require('../runJest');
 
-it('failure messages', () => {
-  const dir = path.resolve(__dirname, '../failures');
-  const {stdout} = runJest(dir);
+const dir = path.resolve(__dirname, '../failures');
+
+test('matcher mesasges', () => {
+  const {stderr, stdout} = runJest(dir, ['test-error-test.js']);
+
+  expect(
+      stderr
+        .replace(/\s*\(\d*ms\)/g, '')
```

Comment:
i already have a utility to make those things easier, i'll fix it here when it's merged